Training25/Program.cs

```diff
@@ -3,11 +3,37 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T1 branch.
+// Program to convert decimal input to binary and hexadecimal.
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      do {
+         Clear ();
+         Write ("Input: ");
+         if (int.TryParse (ReadLine (), out int num) && num >= 0) {
+            WriteLine ("BIN: " + ConvertToBase (2, num));
+            WriteLine ("HEX: " + ConvertToBase (16, num));
+         } else
+            WriteLine ("Please enter a positive integer!");
+         WriteLine ("Press 'Y' to continue");
+      } while (ReadKey (true).Key == ConsoleKey.Y);
    }
+
+   /// <summary>Returns the string of bits converted from decimal to base number system</summary>
+   public static string ConvertToBase (int numBase, int numVal) {

```

Comment:
```suggestion
   public static string ConvertToBase (int base, int val) {
```
I think val/value/num and base are better descriptors for these variables